genie-common-internal/src/main/java/com/netflix/genie/common/internal/dto/DirectoryManifest.java

```diff
@@ -232,15 +286,17 @@ public long getTotalSizeOfFiles() {
         private final Metadata metadata;
         private final TikaConfig tikaConfig;
         private final boolean checksumFiles;
+        private final Filter filter;
 
         ManifestVisitor(
             final Path root,
             final ImmutableMap.Builder<String, ManifestEntry> builder,
-            final boolean checksumFiles
-        ) throws IOException {
+            final boolean checksumFiles,
+            final Filter filter) throws IOException {
```

Comment:
Usually we put the `) throws` on a separate line in these cases

Reply:
Fixed